web/src/HUD.test.tsx

```diff
@@ -40,44 +43,36 @@ const emptyHUD = () => {
     </MemoryRouter>
   )
 }
-const HUDAtPath = (path: string) => {
-  return (
-    <MemoryRouter initialEntries={[path]}>
-      <HUD history={fakeHistory} interfaceVersion={interfaceVersion} />
-    </MemoryRouter>
-  )
-}
 
 beforeEach(() => {
   Date.now = jest.fn(() => 1482363367071)
 })
 
-it("renders without crashing", () => {
-  const div = document.createElement("div")
-  ReactDOM.render(emptyHUD(), div)
-  ReactDOM.unmountComponentAtNode(div)
-})
-
 it("renders reconnecting bar", async () => {
-  const root = mount(emptyHUD())
-  const hud = root.find(HUD)
-  expect(hud.text()).toEqual(expect.stringContaining("Loading"))
+  let root = emptyHUD()
+  let rootTree = renderIntoDocument(root)
```

Comment:
oh! it looks like this may not return the tree. i'm seeing type errors locally and in ci.